src/get-started/beam-overview.md

```diff
@@ -23,6 +23,8 @@ Beam currently supports the following language-specific SDKs:
 * Java ![Java logo]({{ "/images/logos/sdks/java.png" | prepend: site.baseurl }})
 * Python ![Python logo]({{ "/images/logos/sdks/python.png" | prepend: site.baseurl }})
 
+A Scala ![Scalal logo]({{ "/images/logos/sdks/scala.png" | prepend: site.baseurl }}) interface is also available as [Scio](https://github.com/spotify/scio).
```

Comment:
typo (Scalal) - though it won't be visible except in alt text

Reply:
Thanks. Fixed.